features/inventory/validating_product_quantity_on_checkout_complete.feature

```diff
@@ -0,0 +1,35 @@
+@inventory
+Feature: Validating product quantity on checkout complete
+    In order to be sure that products I buy are available
+    As a Customer
+    I want to have them validated on checkout complete
```

Comment:
I want to be prevented from placing an order with products that are out of stock